examples/006_filter_composite_data_example.py

```diff
@@ -82,23 +121,45 @@
 
 composite_model.get_mesh().plot(result_field)
 
+
 # %%
-# List analysis plies
-# ~~~~~~~~~~~~~~~~~~~
-# List all available analysis plies.
-all_ply_names = get_all_analysis_ply_names(composite_model.get_mesh())
-all_ply_names
+# Filtering by material
```

Comment:
```suggestion
# Filter by material
```
Use headings with simple verb forms, not gerunds, which are verbs with "-ing" endings.